src/__mocks__/articles.js

```diff
@@ -0,0 +1,22 @@
+export default (Object.articles = [
+  {
+    author: { username: 'nivelo', bio: null, image: null },
+    body: 'body',
+    coverUrl: 'v1562366643/ninjas/ezzmjn0ak9b5gyzuzo6j.png',
+    createdAt: '2019-07-05T22:43:57.384Z',
```

Comment:
ideally `createdAt` should be populated by the ORM

Reply:
I just used it to mock an article. It is only for testing